autoPyTorch/pipeline/components/training/trainer/RowCutMixTrainer.py

```diff
@@ -0,0 +1,60 @@
+import typing
+
+import numpy as np
+
+import torch
+
+from autoPyTorch.pipeline.components.training.trainer.base_trainer import BaseTrainerComponent
+from autoPyTorch.pipeline.components.training.trainer.mixup_utils import MixUp
+
+
+class RowCutMixTrainer(MixUp, BaseTrainerComponent):
+
+    def data_preparation(self, X: np.ndarray, y: np.ndarray,
+                         ) -> typing.Tuple[np.ndarray, typing.Dict[str, np.ndarray]]:
+        """
+        Depending on the trainer choice, data fed to the network might be pre-processed
+        on a different way. That is, in standard training we provide the data to the
+        network as we receive it to the loader. Some regularization techniques, like mixup
+        alter the data.
+
+        Args:
+            X (np.ndarray): The batch training features
+            y (np.ndarray): The batch training labels
+
+        Returns:
+            np.ndarray: that processes data
+            typing.Dict[str, np.ndarray]: arguments to the criterion function
+        """
+        beta = 1.0
+        lam = self.random_state.beta(beta, beta)
+        batch_size = X.size()[0]
+        index = torch.randperm(batch_size).cuda() if X.is_cuda else torch.randperm(batch_size)
+
+        r = self.random_state.rand(1)
+        if beta <= 0 or r > self.alpha:
```

Comment:
Is `beta` typo of `self.beta` or something? 
```suggestion
        if r > self.alpha:
```

Reply:
It does not seem like that,  if you look at page 3: https://arxiv.org/pdf/1905.04899.pdf
But the check is not necessary you are right, it can be removed.